Apps/Examples/Examples/All Examples/CustomAnchoredSymbolExample.swift

```diff
@@ -0,0 +1,321 @@
+import UIKit
+import MapboxMaps
+import Turf
+
+private enum SymbolTailPosition: Int {
+    case left
+    case right
+    case center
+}
+
+private struct DebugFeature {
+    var coordinate: CLLocationCoordinate2D
+    var highlighted: Bool
+    var sortOrder: Int
+    var tailPosition: SymbolTailPosition
+    var label: String
+    var imageName: String
+}
+
+@objc(CustomAnchoredSymbolExample)
+public class CustomAnchoredSymbolExample: UIViewController, ExampleProtocol {
+    static let symbols = "custom_symbols"
+
+    internal var mapView: MapView!
+
+    // Configure a label
+    public lazy var label: UILabel = {
+        let label = UILabel(frame: CGRect.zero)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.backgroundColor = UIColor.systemBlue
+        label.layer.cornerRadius = 12.0
+        label.textColor = UIColor.white
+        label.textAlignment = .center
+        label.font = UIFont.boldSystemFont(ofSize: 24.0)
+        return label
+    }()
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Center the map camera over New York City
+        let centerCoordinate = CLLocationCoordinate2D(latitude: 40.7128, longitude: -74.0060)
+        let options = MapInitOptions(cameraOptions: CameraOptions(center: centerCoordinate,
+                                                                  zoom: 15.0))
+
+        mapView = MapView(frame: view.bounds, mapInitOptions: options)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        view.addSubview(mapView)
+
+        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector((mapSymbolTap(sender:))))
+        mapView.addGestureRecognizer(tapGestureRecognizer)
+
+        // Allows the delegate to receive information about map events.
```

Comment:
Use of the word `delegate` in this comment is confusing given that this doesn't really fit the typical delegation pattern on iOS